include/wx/generic/animate.h

```diff
@@ -14,19 +14,17 @@
 #include "wx/bitmap.h"
 
 // ----------------------------------------------------------------------------
-// wxAnimation
+// wxGenericAnimation
 // ----------------------------------------------------------------------------
 
-WX_DECLARE_LIST_WITH_DECL(wxAnimationDecoder, wxAnimationDecoderList, class WXDLLIMPEXP_ADV);
-
-class WXDLLIMPEXP_ADV wxAnimation : public wxAnimationBase
+class WXDLLIMPEXP_ADV wxGenericAnimation : public wxAnimationBase
 {
 public:
-    wxAnimation() {}
-    wxAnimation(const wxString &name, wxAnimationType type = wxANIMATION_TYPE_ANY)
+    wxGenericAnimation() {}
+    wxGenericAnimation(const wxString &name, wxAnimationType type = wxANIMATION_TYPE_ANY)
         { LoadFile(name, type); }
 
-    virtual bool IsOk() const wxOVERRIDE
+    virtual bool IsOk() const 
```

Comment:
Why did `override` disappear? AFAICS it should still be there (and normally its absence should result in clang warnings).

Reply:
Not sure, it may have been part of the refacter that was then missed when things mostly changed back.